json/src/test/java/io/airlift/json/TestJsonModule.java

```diff
@@ -390,7 +389,7 @@ private SuperDuperNameList(String superDuperNameList)
             this(superDuperNameList, null);
         }
 
-        private SuperDuperNameList(String superDuperNameList, Object stopJacksonFromUsingStringConstructor)
+        private SuperDuperNameList(String superDuperNameList, Object ignored)
```

Comment:
Add a comment indicating why that parameter needs to be there. The previous name was the documentation for it. Otherwise, someone might inadvertently remove it.